### ISSUE TITLE
Looking for the maintainer ❤️ 

### ISSUE THREAD

**Issue by jukben** [jukben/keyboardio-led-custom-palette#6]
Because I sold my own keyboard, I can't support this anymore – I can't test it. 

@algernon I can give you a maintainer role, if you would like to - it would be awesome. I've put some hours into this project and it would be great to see it alive. ❤️ 

But anyone, in general, is welcomed!

**Comment by algernon** [comment]
I plan to build something similar into Chrysalis soonish... the original plan was to have a pre-alpha out the door, and then invite interested people - you included - to help build the missing functionality. I guess that's out of the picture now :)

I'll see what I can do to restore the functionality, but I wouldn't feel comfortable maintaining it longer term, I'm afraid. My React, JS, and front-end knowledge in general is minimal, and I'd love to reduce the number of front-end things I maintain, not add more O:)

**Comment by jukben** [comment]
Yeah, that's reasonable. Actually, yeah, I was thinking about to help you (feel free to rip out every part of this project you might need) but as soon I'm not using the keyboard it's not very actual. But it's sad, the ecosystem (Kaleidoscope) was awesome to work with. Thanks for that.

**Comment by algernon** [comment]
FWIW, I added a colormap editor to Chrysalis a few hours ago, borrowing a few ideas from here. Not perfect yet, but it lets you edit the colormap.
